Document setting the coverage reports directory (#2400)
vitest-dev/vitest: docs/guide/coverage.md

```

Please refer to the type definition for more details.

> ## Changing the default coverage folder location

Author: ```suggestion
## Changing the default c8 coverage folder location
```

> ## Changing the default c8 coverage folder location

Author: ```suggestion
## Changing the default coverage folder location
```

> When running a coverage report, a temp folder is created in the root directory. If you want to move it to another location, use the `test.coverage.reportsDirectory` property in the `vite.config.js` file.

Reviewer: > a temp folder is created in the root directory

Only `c8` provider uses the temp directory. `istanbul` doesn't need it.

> When running a coverage report, a `coverage` folder is created in the root directory. If you want to move it to another directory, use the `test.coverage.reportsDirectory` property in the `vite.config.js` file.

Author: ```suggestion
When running a coverage report, a `coverage` folder is created in the root directory of your project. If you want to move it to a different directory, use the `test.coverage.reportsDirectory` property in the `vite.config.js` file.
```

> When running a coverage report with c8, a temp folder is created in the root directory. If you want to move it to another location, use the `test.coverage.reportsDirectory` property in the `vite.config.js` file.

Author: ```suggestion
When running a coverage report, a `coverage` folder is created in the root directory. If you want to move it to another location, use the `test.coverage.reportsDirectory` property in the `vite.config.js` file.
```
